### PR TITLE
Add community hours details to READMEs

### PR DESCRIPTION
**Pull request by jnummelin**
Add community hours call details to READMEs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,14 @@ check out the [Contributing Guide] and the [Code of Conduct].
 [Contributing Guide]: https://docs.k0sproject.io/stable/contributors/overview/
 [Code of Conduct]:https://docs.k0sproject.io/stable/contributors/CODE_OF_CONDUCT/
 
+### Community hours
+
+We will be holding regularly scheduled community hours, where anyone in the community is welcome to drop in and ask us questions, talk about projects, and chat.
```
Comment (twz123) [inline README.md:74]: I wonder if we could even drop "in the community" :thinking:

```suggestion
We will be holding regular community hours. Everyone in the community is welcome to drop by and ask questions, talk about projects, and chat.
```

File: README.md [hunk 2]
```diff
@@ -69,6 +69,14 @@ check out the [Contributing Guide] and the [Code of Conduct].
...
+
+We currently hold the office hours call monthly, on last Tuesday of the month.
```
Comment (twz123) [inline README.md:76]:
```suggestion
We currently have a monthly office hours call on the last Tuesday of the month.
```

File: README.md [hunk 3]
```diff
@@ -69,6 +69,14 @@ check out the [Contributing Guide] and the [Code of Conduct].
...
+
+To see the call details in your local timezone, check out [https://dateful.com/eventlink/2735919704](https://dateful.com/eventlink/2735919704).
```
Comment (twz123) [inline README.md:78]:
```suggestion
To see the call details in your local timezone, check out the [event on Dateful](https://dateful.com/eventlink/2735919704).
```

**Review by twz123 — APPROVED** [review]
Looking forward to the 30th.

**Comment by jnummelin** [comment]
@twz123 added your suggestions, PTAL

**Review by twz123 — APPROVED** [review]

**Pull request merged**